larq/layers.py

```diff
@@ -1321,3 +1333,46 @@ def __init__(
             implementation=implementation,
             **kwargs
         )
+
+
+@utils.register_keras_custom_object
+class QuantDepthwiseConv2D(QuantizerBase, tf.keras.layers.DepthwiseConv2D):
+    def __init__(
+        self,
+        kernel_size,
+        strides=(1, 1),
+        padding="valid",
+        depth_multiplier=1,
+        data_format=None,
+        activation=None,
+        use_bias=True,
+        input_quantizer=None,
+        kernel_quantizer=None,
```

Comment:
I think this should be called `depthwise_quantizer` to be consistent with `depthwise_regularizer` and `depthwise_constraint`, `depthwise_initializer`.